rpmconf/Install/zmsetup.pl

```diff
@@ -7102,6 +7108,9 @@ sub applyConfig {
     # 32295
     setLdapGlobalConfig("zimbraSkinLogoURL", "http://www.zimbra.com")
```

Comment:
FWIW, this url really should use https

Reply:
Let's open another ticket to address this since that is not part of this change.

Reply:
ZCS-4451 was created to address this.